### PR TITLE
Add 21st Real Estate

### PR DESCRIPTION
**Pull request by aloise**
## Add 21st Real Estate

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [ ] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

**Comment by poteto** [comment]
Can you move this down to the Y - # section?

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Also check out [No Whiteboards](https://nowhiteboards.io) to search for jobs at
 ---
 
 ## 0 - 9
+- [21st Real Estate](https://21re.de) | Berlin, Germany | Phone call for quick personal introduction followed by Video call interview. Finally, a pair-programming session on-site.
```
Comment (poteto) [inline README.md:35]: Can you move this down to the Y - # section instead?

Comment (aloise) [reply README.md]: Sure!

**Pull request merged**